includes/class-airtable-api.php

```diff
@@ -2,140 +2,153 @@
 
 namespace UbcVpfoSpacesPage;
 
-defined('ABSPATH') || exit;
+defined( 'ABSPATH' ) || exit;
 
 use TANIOS\Airtable\Airtable;
 
-class Airtable_Api
-{
+class Airtable_Api {
 
-    public Airtable $airtable;
-    public string $site_location;
-    private $van_airtable;
-    private $okan_airtable;
+	public Airtable $airtable;
+	public string $site_location;
+	private $van_airtable;
+	private $okan_airtable;
 
-    const CACHE_TTL = 3600;
-    const ROOMS_PER_PAGE = 10;
+	const CACHE_TTL      = 3600;
+	const ROOMS_PER_PAGE = 10;
 
-    const LOCATION_VAN = 'van_airtable';
-    const LOCATION_OKAN = 'okan_airtable';
+	const LOCATION_VAN  = 'van_airtable';
+	const LOCATION_OKAN = 'okan_airtable';
 
-    public $cache_prefix = 'airtable_cache_';
+	public $cache_prefix = 'airtable_cache_';
 
-    private static $campus_mapping = array(
-        'vancouver' => 'van_airtable',
-        'okanagan'  => 'okan_airtable',
-    );
+	private static $campus_mapping = array(
+		'vancouver' => 'van_airtable',
+		'okanagan'  => 'okan_airtable',
+	);
 
-    public function __construct()
-    {
-        $this->airtable = $this->negotiateAirTableLocation();
-    }
+	public function __construct() {
+		$this->airtable = $this->negotiate_air_table_location();
+	}
 
-    public function get_building_by_slug(string $building_slug)
-    {
-        $params = array(
-            'filterByFormula' => sprintf("AND( slug = '%s' )", $building_slug),
-            'maxRecords'      => 1,
-        );
+	public function get_building_by_slug( string $building_slug ) {
+		$params = array(
+			'filterByFormula' => sprintf( "AND( slug = '%s' )", $building_slug ),
+			'maxRecords'      => 1,
+		);
 
-        $response = $this->get(table: 'Buildings', params: $params, resource: $building_slug);
+		$response = $this->get( table: 'Buildings', params: $params, request_resource: $building_slug );
```

Comment:
Just a note that the codesniffer for WP Extra really did not like using "resource" so I changed it.

Reply:
strange, this is a named paramiter, I'm not sure if you've seen them before.

With older PHP you'd have to use the method properties in the order they are defined in the method.

`sample function( $one, $two = null, $three)`

With a named property you can skip 

`sample(sample: 'something', one: 'something')`

Reply:
Cool! The codesniffer scan just told me not to use "resource" because WP wants to keep it reserved to avoid confusion or something? They prob use it in core somewhere